### PR TITLE
🎇Add SparkHacks 2024

### PR DESCRIPTION
**Pull request by SohamG**
- New deployment for container
- New Service for deployment
- Point existing IngressRoute to new Service

NOT TESTED YET!

### CODE REVIEW

File: kubernetes/argocd/stacks/sparkhacks/sparkhacks-2024.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      app.kubernetes.io/name: sparkhacks-2024
+  template:
+    metadata:
+      annotations:
+      labels:
+        app.kubernetes.io/name: sparkhacks-2024
+    spec:
+      containers:
+        - name: sparkhacks-2024
+          image: "docker pull ghcr.io/sparkhacks/sparkweb:main"
```
Comment (clee231) [inline kubernetes/argocd/stacks/sparkhacks/sparkhacks-2024.yml:20]:
```suggestion
          image: "ghcr.io/sparkhacks/sparkweb:main"
```

File: kubernetes/argocd/stacks/sparkhacks/sparkhacks-2023.yml [hunk 2]
```diff
@@ -68,7 +68,7 @@ spec:
       match: "Host(`sparkhacksatuic.com`) || Host(`www.sparkhacksatuic.com`)"
       services:
         - kind: Service
-          name: sparkhacks-2023
+          name: sparkhacks-2024 # Perhaps move to new file.
```
Comment (clee231) [inline kubernetes/argocd/stacks/sparkhacks/sparkhacks-2023.yml:71]: Yeah, we can move this to a different file. I was thinking when we switch, I'd add a `/2023` IngressRoute, so the old site can still be accessible.

**Review by clee231 — CHANGES_REQUESTED** [review]

File: kubernetes/argocd/stacks/sparkhacks/sparkhacks-2024.yml (hunk 1 above)
Comment (SohamG) [reply kubernetes/argocd/stacks/sparkhacks/sparkhacks-2024.yml:20]: uuuuuguuguguuuuuuuughgggggggggghhhhhhhhhhhhhhhhhhhhh

**Comment by SohamG** [comment]
kubelint still broken pepehands

**Review by clee231 — APPROVED** [review]
Looks good to me!
image

**Pull request merged**